File: python/python-print-invert-triangle/example2.py
# https://codevscolor.com/python-print-invert-triangle

height = int(input("Enter the height of the triangle: "))
c = str(input("Enter the character you want to print the triangle: "))

for i in range(0, height):
    for j in range(0, height - i):
        print(c + " ", end="")
    print()

File: C/c-print-alternate-numbers-array/example2.c
// https://codevscolor.com/c-print-alternate-numbers-array
#include <stdio.h>

int main()
{
    int i;
    int size;

    printf("How many numbers you want to enter: ");
    scanf("%d", &size);

    int array[size];

    i = 0;
    while (i < size)
    {
        printf("Enter number %d: ", i + 1);
        scanf("%d", &array[i]);
        i++;
    }

    printf("Alternate elements of the input array: \n");
    i = 0;
    while (i < size)
    {
        printf("%d ", array[i]);
        i += 2;
    }
}